core/src/main/scala/org/bykn/bosatsu/pattern/SetOps.scala

```diff
@@ -123,6 +172,29 @@ object SetOps {
       def subset(a: A, b: A): Boolean = ordA.equiv(a, b)
     }
 
+  def fromFinite[A](items: Iterable[A]): SetOps[Set[A]] =
```

Comment:
this was important to better model the Pattern matching we have with enums. It immediately exposed a bug.